backend/database/wrapper/field_wrapper.py

```diff
@@ -41,12 +41,18 @@ def get_field_name(self) -> str:
 
 def get_explanations(dynamic_field_list) -> (Dict[str, FieldExplanation], List[FieldExplanation]):
     field_list = [
-        FieldExplanation('assists', 'Total number of passes that led to goals.'),
-        FieldExplanation('shots', 'Total number of hits towards the enemy goal.'),
+        FieldExplanation('goals', 'Total number of goals scored by the player obtained from the match data.'),
+        FieldExplanation('saves', 'Total number of saves by the player obtained from the match data.'),
+        FieldExplanation('score', 'In-game player score for the match obtained from the match data.'),
+        FieldExplanation('assists', 'Total number of passes that led to goals obtained from the match data.'),
+        FieldExplanation('shots', 'Total number of hits towards the enemy goal obtained from the match data.'),
         FieldExplanation('total_passes', 'Total hits followed by a teammate hit.', field_rename='passes'),
         FieldExplanation('total_hits', 'Total number of hits (using hit detection).',
                          field_rename='hits'),
-        FieldExplanation('total_dribble_conts ', 'The second, third, … touch of a dribble.'),
+        FieldExplanation('total_goals', 'Total number of goals used for hit analysis.'),
+        FieldExplanation('total_saves', 'Total number of saves used for hit analysis.'),
+        FieldExplanation('total_shots', 'Total number of shots used for hit analysis.'),
```

Comment:
We should make sure to clarify that this is our definition of shots which is different than psyonix definition.   Our definition is any hit that will be inside the goal within 5 seconds using ball prediction assuming that no other player will touch it

Reply:
Interesting, will clarify it in the explanation.